tasks/app/initialize.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring
+"""
+This file contains initialization data for development usage only.
+
+You can execute this code via ``invoke app.consistency``
```

Comment:
should this be app.initialize?

Reply:
I have updated the text here and also added a description to this task that specifies that the data is coming from the EDM